src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greet = showWelcomeScreen();
+        System.out.println(greet);
+        
+        Task currentTask;
+        Task[] storeTasks = new Task[100];
+        int numOfTasks = 0;
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            line = in.nextLine();
+            if (line.equals("bye")) {
+                break;
+            }
+
+            if (line.equals("list")) {
+                System.out.println(" ____________________________________________________________");
+                System.out.println("  Here are the tasks in your list:");
+                for (int i = 0; i < numOfTasks; i++) {
+                    System.out.println("  " + (i + 1) + "." + storeTasks[i]);
+                }
+                System.out.println(" ____________________________________________________________");
+                continue;
+            }
+
+            if (line.contains("done")) {
+                String[] word = line.split(" ");
+                int taskNum = Integer.parseInt(word[1]);
+                if (taskNum >= 1 || taskNum <= numOfTasks) {
```

Comment:
I like the way of judging the conditions of "done" command!

Reply:
Thank you!